### PR TITLE
Expose pluralize helper behavior

### LINKED ISSUE #122: Expose helper behavior as an importable function
I started using this library after a while of not using it and was surprised that I couldn't get similar output from

```hbs
{{pluralize 2, 'cat'}} {{!-- "2 cats" --}}
```

and

```js
import { pluralize } from 'ember-inflector';

//...
return pluralize(2, 'cat'); // throws an exception
//...
```

Instead the function only accepts a single string argument.

There are obviously backwards-compatibility issues to worry about here. Perhaps we should export a `pluralize` function from `ember-inflector/lib/helpers/pluralize` that behaves like the helper (and same for `singularize`)?

Happy to work on this if I get confirmation on the direction.

### PR DESCRIPTION
**Pull request by juanazam**
closes #122

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ Inflector.inflector.pluralize("taco"); // tacos
 singularize("tacos"); // taco
 pluralize("taco"); // tacos
 
+pluralize(2, "taco"); // 2 tacos
+pluralize(2, "tacos", { without-count: true }); // tacos
```
Comment (stefanpenner) [inline README.md]: this isn't valid JS `'without-count'` or `withoutCount`

**Comment by juanazam** [comment]
@stefanpenner changed the implementation a bit because the exported version was not respecting the same signature as the template version.

```js
  pluralize(2, "word", { withoutCount: true });
```

```hbs
  {{pluralize 2 "word" without-count=true}}
```

**Comment by juanazam** [comment]
Hey @stefanpenner, any news on this?

**Comment by amiel** [comment]
I'd like to bump this. I would definitely find it useful.

**Comment by stefanpenner** [comment]
@amiel  / @juanazam  ya looks good, but requires some tests before we can merge/release.

**Comment by jamesarosen** [comment]
This PR doesn't have tests and doesn't work with caching enabled. I'm on it.

**Comment by stefanpenner** [comment]
released as v2.2.0 🎉

**Pull request closed without merging**